app/src/index.html

```diff
@@ -6,6 +6,7 @@
   <link rel="stylesheet" href="/reset.css" type="text/css" media="screen">
 </head>
 <body>
+  <div id="portal"></div>
```

Comment:
Is it significant that this one comes before app?

Reply:
Yes, because otherwise it would be beneath of whatever "app" renders and we would have to force it above manually by setting `zIndex`. I rather prefer the order of the markup to reflect what we see on screen.

Reply:
Great, maybe inline a comment about it.